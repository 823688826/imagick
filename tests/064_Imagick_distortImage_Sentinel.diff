001+ unknown method given: No such file or directory
001- Ok
002+ 
003+ Fatal error: Uncaught exception 'ImagickException' with message 'memory allocation failed GenerateCoefficients @ error/distort.c/GenerateCoefficients/497' in /home/github/mkoppanen-imagick/tests/064_Imagick_distortImage_Sentinel.php:19
004+ Stack trace:
005+ #0 /home/github/mkoppanen-imagick/tests/064_Imagick_distortImage_Sentinel.php(19): Imagick->distortimage(18, Array, true)
006+ #1 {main}
007+   thrown in /home/github/mkoppanen-imagick/tests/064_Imagick_distortImage_Sentinel.php on line 19