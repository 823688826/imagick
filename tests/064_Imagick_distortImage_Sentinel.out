unknown method given: No such file or directory

Fatal error: Uncaught exception 'ImagickException' with message 'memory allocation failed GenerateCoefficients @ error/distort.c/GenerateCoefficients/497' in /home/github/mkoppanen-imagick/tests/064_Imagick_distortImage_Sentinel.php:19
Stack trace:
#0 /home/github/mkoppanen-imagick/tests/064_Imagick_distortImage_Sentinel.php(19): Imagick->distortimage(18, Array, true)
#1 {main}
  thrown in /home/github/mkoppanen-imagick/tests/064_Imagick_distortImage_Sentinel.php on line 19